server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/ScanDataSource.java

```diff
@@ -238,10 +238,16 @@ private SortedKeyValueIterator<Key,Value> createIterator() throws IOException {
   public void close(boolean sawErrors) {
 
     if (memIters != null) {
+      log.debug("Returning mem iterators");
```

Comment:
Logging debug per scan would be a lot of logging.  I would put these two debugs at trace.  Would also be good to include something that identifies the scan, like the extent.